src/GShark/Geometry/Point3.cs

```diff
@@ -717,7 +717,26 @@ public int InPolygon(Polygon polygon)
             }
             return inside ? 1 : -1;
         }
-        
+
+
+
+        //Function that finds point in a cloud of points to a target point
+        public Point3 CloudClosestPoint(List<Point3> cloud)
+        {
+            double Min_list = double.MaxValue;
```

Comment:
Why named `Min_list`?
Rename to `minValue` or `minDistance`, use camelCase.

```suggestion
            double minDistance= double.MaxValue;
```